tools/walletextension/README.md

```diff
@@ -27,19 +27,25 @@ unencrypted. If the data is not particularly sensitive, it can also be run in an
 
 ## Usage
 
-1. Clone this repository: https://github.com/obscuronet/go-obscuro
+1. Precompiled binaries for the wallet extension are available for macOS AMD64 and Windows AMD64 on the project's 
+   [releases page](https://github.com/obscuronet/go-obscuro/releases/)
 
-2. Build the wallet extension binary by running the following from the root of the repository:
-  ```
-  cd tools/walletextension/main
-  go build -o wallet_extension .
-  ```
-  This will build the wallet extension binary under `tools/walletextension/main/wallet_extension`
+   * For other OSes and architectures, the wallet extension binary can be built by cloning the 
+     [Obscuro repository](https://github.com/obscuronet/go-obscuro) and running the following commands from the root of 
+     the repository:
+     
+     ```
+     cd tools/walletextension/main
+     go build -o wallet_extension .
+     ```
+     
+     This will build the wallet extension binary under `tools/walletextension/main/wallet_extension`
 
-3. Run the `wallet_extension` binary to start the wallet extension:
-  ```./wallet_extension --nodeRPCHTTPAddress=<Obscuro host RPC HTTP address> --nodeRPCWebsocketAddress=<Obscuro host RPC websocket address>```
+3. Start the wallet extension by running the `wallet_extension` binary with the following flags:
```

Comment:
Need to update the numbering on these points I think

Reply:
Fixed